tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false"># From list
&gt;&gt;&gt; in this case, what is the default dtype and device? 
&gt;&gt;&gt; How is it determined?
a = o3d.core.Tensor([0, 1, 2])
print(a)
&gt;&gt;&gt; Consider doing <code>print("Created from list:\n{}\n".format(a))</code>
&gt;&gt;&gt; This way the user can look at the outputs and know who printed
&gt;&gt;&gt; the tensor. Same for all the print statements below.


# Dtype inferred from list.
a_float = o3d.core.Tensor([0.0, 1.0, 2.0])
print(a_float)


# From Numpy
&gt;&gt;&gt; Mention this is a copy, rather than shared memory.
a = o3d.core.Tensor(np.array([0, 1, 2]))
print(a)


# Specify dtype
a = o3d.core.Tensor(np.array([0, 1, 2]), dtype=o3d.core.Dtype.Float64)
print(a)


# Specify device
# Specify dtype
&gt;&gt;&gt; remove "# Specify dtype"
a = o3d.core.Tensor(np.array([0, 1, 2]), device=o3d.core.Device("CUDA:0"))
print(a)

&gt;&gt;&gt; Add the copy constructor case
&gt;&gt;&gt; What happends to the underlying memory? 
&gt;&gt;&gt; (actually they are "shallow copy", so do a demo of shallow copy,
&gt;&gt;   by changing one value of a and show that b is also changed.)
a = o3c.Tensor(...)
b = o3c.Tensor(a)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>